Misc/NEWS.d/next/Tests/2019-03-14-14-50-11.bpo-31904.iu3g9k.rst

```diff
@@ -0,0 +1 @@
+Skip ExecTests and URandomFDTests in test_os.py for VxWorks.
```

Comment:
Honestly, I don't think that this NEWS entry is really helpful to users. Can you please remove it?

Reply:
Removed.